src/wh_client.c

```diff
@@ -119,66 +120,117 @@ int wh_Client_Cleanup(whClient* c)
     return 0;
 }
 
-int wh_Client_EchoRequest(whClient* c, uint16_t size, const void* data)
+int wh_Client_SendRequest(whClient* c,
```

Comment:
null check on `c` (only `c->comm` is `NULL` checked in `wh_CommClientSendRequest()`)

Reply:
fixed.